back/lib/finder/sortable.rb

```diff
@@ -52,7 +52,7 @@ def _sort_scopes
     private
 
     def _sort_records
-      @_sort_method = params.delete(:sort) || _default_sort
+      @_sort_method = params[:sort] || _default_sort
```

Comment:
This was deleting the sort param from action controller params, making it hard to debug.